src/transformers/models/whisper/english_normalizer.py

```diff
@@ -18,10 +18,17 @@
 from fractions import Fraction
 from typing import Iterator, List, Match, Optional, Union
 
-from more_itertools import windowed
+from transformers.utils.import_utils import is_more_itertools_available
```

Comment:
For consistency, please add the is_available function to the init file of the utils folder, to make sure it can be imported from the utils directory

Reply:
This doesn't seem to be resolved

Reply:
Ok update: saw you added it to the utils folder, but you can also update the import here ;)

Reply:
Also, use a relative import:
```suggestion
from .utils import is_more_itertools_available
```